tests/functionaltests/Tests/UIKitTests/UIButtonTests.mm

```diff
@@ -51,75 +36,98 @@
     ASSERT_TRUE([backingElement isKindOfClass:[WXFrameworkElement class]]);
 }
 
-TEST(UIButton, BackgroundColorChanged) {
-    __block UIButtonViewController* buttonVC;
+TEST(UIButton, TitleColorChanged) {
+    __block BOOL signaled = NO;
     __block NSCondition* condition = [[[NSCondition alloc] init] autorelease];
```

Comment:
>autorelease [](start = 65, length = 11)

nit: I think it would be nicer to avoid the autorelease pool for all of these, so we have more deterministic cleanup.